cmake/Translation.cmake

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+set(LY_I18N_BUILD OFF CACHE BOOL "I18N generators")
+set(LY_I18N_LANGUAGE "en_US" CACHE STRING "I18N target language")
```

Comment:
The prefix of "O3DE" should be used instead of "LY", as "LY" is left over from Lumberyard and newer variables are prefixed with "O3DE" to represent the name of the engine.

Also can you add a comment that mentions that "i18n" stands for "Internationalization" as not all developers might not know what "i18n" stands for?

Reply:
I'm also concerned about using a CMAKE variable for the I18N language at all.  It being a CMake variable would indicate that in some way it is baked into the executable.  However, we don't really want to recompile executables and binaries in order to switch languages, right?  Should it not be a runtime option, with all language QMs built?

Reply:
Actually discount my above comment, looksl ike this is the _array_ of languages _to build_
Perhaps the comment should be "List of languages to include in the build" rather than "target language" ?